atlas/pixi.toml

```diff
@@ -0,0 +1,19 @@
+[project]
+name = "atlas-integration-challenge"
+version = "0.0.1"
+description = "IRIS-HEP ATLAS integration challenge"
+authors = ["Alexander Held <alexander.held@cern.ch>"]
+channels = ["conda-forge"]
+platforms = ["osx-arm64"]
```

Comment:
@alexander-held To make this x86 Linux compliant as well run

```
pixi workspace platform add linux-64
```

which will make this line

```
platforms = ["osx-arm64", "linux-64"]
```

and update the lock file.

Reply:
Thanks! Will add this.